fedmsg.d/sse.py

```diff
@@ -9,6 +9,24 @@
     "fmn.sse.webserver.tcp_port": 8080,
     "fmn.sse.webserver.log": "sse.log",
 
+    # A regular expression using the standard Python re syntax that defines a
+    # whitelist of queues exposed by the SSE server.
+    'fmn.sse.webserver.queue_whitelist': '.+\.id\.fedoraproject\.org$',
+
+    # A regular expression using the standard Python re syntax that defines a
+    # blacklist for queues exposed by the SSE server. Any queue name that is
+    # matched by the regular expression will return a HTTP 403 to the client.
+    #
+    # Note: This is applied _after_ the whitelist so if the queue is matched
+    # by both regular expressions, the queue _will not_ be served.
+    'fmn.sse.webserver.queue_blacklist': None,
```

Comment:
if we have a whitelist isnt a blacklist not necessary? One case where two of them would be useful is we have spammers who pass the whitelist and we need to specifically block their queues.